docs/ref.md

```diff
@@ -64,10 +70,12 @@ There is currently no special mechanism in place for controlling banners.
     -->
     ```
 
+
 ## Glossary
+
 There is a glossary of HPC-related acronyms that get highlighted automatically. For example: CPU,
 GPU, QPU, etc. The acronyms are defined in the markdown file
-`csc-overrides/assets/glossaries/hpc.md`. More acronyms (case-sensitive) can be added there or into another
+_csc-overrides/assets/snippets/glossaries/hpc.md_. More acronyms (case-sensitive) can be added there or into another
```

Comment:
What's our current policy on using inline code blocks? Shouldn't paths be formatted as code?

Reply:
Perhaps this should have been discussed. :thinking: Anyway, my opinion is to leave the inline and fenced code specifically for examples of actual code or commands. When referencing a name, even when the name usually appears in some code (and even when it happens to be prefixed with a path), I would use something else to format it. For example:

The static method _Classname.static_method_ is used like this:
```python
Classname.static_method(arg)
```

The file _~/scripts/script.sh_ needs to have the execution permission set:
```bash
chmod u+x ~/scripts/script.sh
```

I'm actually not sure whether boldface would be even better. In any case, looks like I'm not even abiding by my own rules in the Snippets section, where I've used (or probably forgot to reformat) inline code.